profiler/src/profiler/TracyView_Timeline.cpp

```diff
@@ -10,6 +10,14 @@
 #include "TracyTimelineItemThread.hpp"
 #include "TracyView.hpp"
 
+#if defined _WIN32 && (defined __GNUC__ && !defined __clang__)
+//_WIN32 - only tested on windows
+
+// gcc throws error for not present std::pow function,
+// clang does not have that problem
+#include <cmath>
+#endif
```

Comment:
The correct solution would be to change `std::pow` to `pow` and include `math.h` (with no platform checks).